baseapp/router.go

```diff
@@ -36,7 +36,7 @@ var isAlpha = regexp.MustCompile(`^[a-zA-Z]+$`).MatchString
 // AddRoute - TODO add description
 func (rtr *router) AddRoute(r string, h sdk.Handler) Router {
 	if !isAlpha(r) {
-		panic("route expressions can only contain alphanumeric characters")
+		panic("route expressions can only contain alphabet characters")
```

Comment:
Good catch! However I think the regex not allowing numerics is the bug here, not the error message being mismatched.

Reply:
Let's open a new issue for this